### PR TITLE
Fix binaryen path

### PR DESCRIPTION
**Pull request by Ludea**

### CODE REVIEW

File: packages/binaryen/build.sh [hunk 1]
```diff
@@ -2,5 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://github.com/WebAssembly/binaryen
 TERMUX_PKG_DESCRIPTION="Binaryen is a compiler and toolchain infrastructure library for WebAssembly"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_VERSION=98
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://github.com/WebAssembly/binaryen/archive/version_${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=9f805db6735869ab52cde7c0404879c90cf386888c0f587e944737550171c1c4
+
+termux_step_post_make_install () {
+      export EM_BINARYEN_ROOT=$TERMUX_PREFIX/usr/bin
```
Comment (Grimler91) [inline packages/binaryen/build.sh:10]: Please use tab instead of spaces

**Review by Grimler91 — CHANGES_REQUESTED** [review]

File: packages/binaryen/build.sh (hunk 1 above)
Comment (xeffyr) [reply packages/binaryen/build.sh:10]: I'm more curious about the reason of setting environment variable in **post installation** step. And why using `$TERMUX_PREFIX/usr/bin` btw?

* Setting any variable in post_make_install is useless.
* Termux doesn't use `$PREFIX/usr`. `bin` directory must be exactly under prefix.

Spaces vs tabs is another question here.

**Review by xeffyr — CHANGES_REQUESTED** [review]

File: packages/binaryen/build.sh (hunk 1 above)
Comment (Ludea) [reply packages/binaryen/build.sh:10]: switching spaces to tabs, tabs doesn't work on my phone

Comment (Ludea) [reply packages/binaryen/build.sh:10]: > Setting any variable in post_make_install is useless.
 I need to install package, and if install is successful, set binaries env path. Maybe I'm wrong on the step

> Termux doesn't use $PREFIX/usr. bin directory must be exactly under prefix.
$TERMUX_PREFIX = /data/data/com.termux/files, right ? I need to specify the binary full path, ie /data/data/com.termux/files/usr/bin


.

Comment (xeffyr) [reply packages/binaryen/build.sh:10]: > The env variable is for emscripten package

Then it should be set in `emscripten` build.sh.

These steps *are not executed* once package is built and as result will not work if binariyen and emscripten are built in separate tasks

.> $TERMUX_PREFIX = /data/data/com.termux/files, right ?

Termux uses `/data/data/com.termux/files/usr` as "rootfs" or prefix. Home is stored outside. So:

* `/data/data/com.termux/files` - basedir, app-managed. Should never be used as package installation root.
* `/data/data/com.termux/files/home` - home, user-managed.
* `/data/data/com.termux/files/usr` - prefix/rootfs, managed by `apt`.

**Comment by Ludea** [comment]
Closing,, in favor of https://github.com/termux/termux-packages/pull/6111

**Pull request closed without merging**